### ISSUE TITLE
automatically mirror after merge to main

### ISSUE THREAD

**Issue by christianguenter2** [abap2UI5/abap2UI5#1089]
Now that we have the [mirror](https://github.com/christianguenter2/abap2UI5-mirror.git), it would be nice to automatically mirroring after a pr is merged. I think that's something which should be possible with gh actions somehow. Unfortunately that's something way over my head.

**Comment by oblomov-dev** [comment]
yes i can put it on my list, just calling an already setup npm command via github action should be (hopefully) possible for me :)

but maybe the action should be setup in the mirror repository and we transfer this issue to there?

**Comment by larshp** [comment]
possible, but I dont think it would be the typical use-case

if someone wants to mirror they setup their own repo, with the name they want, this repo might be public or private. Thus its easier to have a cron job in the mirror pulling the code from the source

**Comment by oblomov-dev** [comment]
thanks for the info, ok when i find time i will give it a try in the mirror repo with a cron job. it makes somehow no sense, but then this repo is a good copy&paste template for everyone who needs the functionality in the future.

**Comment by christianguenter2** [comment]
`in the mirror repository and we transfer this issue to there?`  - makes sense. 
Seems that I'm not allowed to do that. So feel free.
